app/controllers/organizations/staff/external_form_upload_controller.rb

```diff
@@ -11,8 +11,15 @@ def index
 
       def create
         authorize! :external_form_upload, context: {organization: Current.organization}
-        import = Organizations::Importers::CsvImportService.new(params[:files]).call
-        render turbo_stream: turbo_stream.replace("results", partial: "organizations/staff/external_form_upload/upload_results", locals: {import: import})
+        file = params[:files]
+        @blob = ActiveStorage::Blob.create_and_upload!(io: file, filename: file.original_filename)
+
+        CsvImportJob.set(wait: 1.second).perform_later(@blob.signed_id, Current.organization.id)
```

Comment:
I don't love this wait but there was an issue due to timing. If an exception was raised (in the validation_file method) the `blob.signed_id` would not match the id created in the view. The view needs the broadcast channel to receive the message that the import is complete. 

I need to see if using something other than the blob.signed_id will be sufficient for the pub/sub channel.